data/Tanmay-Joshi73.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Tanmay Joshi",
+  "displayStatsPublic":true,
+  "type":"personal",
+  "bio": "Passionate student on a quest to master the art of software development. 🚀",
+  "avatar":"https://github.com/Tanmay-Joshi73.png",
```

the following are no longer necessary:
```suggestion
  "bio": "Passionate student on a quest to master the art of software development. 🚀",
```